### PR TITLE
resume fix

### PR DESCRIPTION
**Pull request by Razvan-Morar**
don't advance position on interrupt, only on resume

### CODE REVIEW

File: cloudslang-runtime/src/main/java/io/cloudslang/lang/runtime/steps/ExecutableExecutionData.java [hunk 1]
```diff
@@ -183,20 +183,23 @@ public void startExecutable(@Param(ScoreLangConstants.EXECUTABLE_INPUTS_KEY) Lis
...
             }
 
+            boolean cont = true;
             if (systemContext.containsKey(ScoreLangConstants.USER_INTERRUPT)) {
                 Long parentRid = null;
                 if (systemContext.containsKey(PARENT_RUNNING_ID)) {
                     parentRid = (Long) systemContext.get(PARENT_RUNNING_ID);
                 }
-                this.debuggerBreakpointsHandler.handleBreakpoints(
+                cont = !this.debuggerBreakpointsHandler.handleBreakpoints(
```
Comment (ilasalexandra) [inline cloudslang-runtime/src/main/java/io/cloudslang/lang/runtime/steps/ExecutableExecutionData.java:192]: 'this' is redundant here

File: cloudslang-runtime/src/main/java/io/cloudslang/lang/runtime/steps/ExecutableExecutionData.java [hunk 2]
```diff
@@ -183,20 +183,23 @@ public void startExecutable(@Param(ScoreLangConstants.EXECUTABLE_INPUTS_KEY) Lis
                 }
             }
 
+            boolean cont = true;
```
Comment (alexbajzat) [inline cloudslang-runtime/src/main/java/io/cloudslang/lang/runtime/steps/ExecutableExecutionData.java:186]: Please use full names. I'm not sure what this refers to.

File: cloudslang-runtime/src/main/java/io/cloudslang/lang/runtime/steps/ExecutableExecutionData.java [hunk 3]
```diff
@@ -183,20 +183,23 @@ public void startExecutable(@Param(ScoreLangConstants.EXECUTABLE_INPUTS_KEY) Lis
...
-                this.debuggerBreakpointsHandler.handleBreakpoints(
+                cont = !this.debuggerBreakpointsHandler.handleBreakpoints(
                         systemContext,
                         runEnv,
                         executionRuntimeServices,
                         stepType,
                         nodeName,
-                        executionRuntimeServices.extractParentNameFromRunId(parentRid) + "." + nodeName);
+                        executionRuntimeServices.extractParentNameFromRunId(parentRid) + "." +
+                                executionRuntimeServices.getNodeName());
```
Comment (alexbajzat) [inline cloudslang-runtime/src/main/java/io/cloudslang/lang/runtime/steps/ExecutableExecutionData.java:199]: why do we need to fetch the nodeName again? Is it different?

**Review by alexbajzat — APPROVED** [review]

File: cloudslang-runtime/src/main/java/io/cloudslang/lang/runtime/steps/ExecutableExecutionData.java (hunk 3 above)
Comment (alexbajzat) [reply cloudslang-runtime/src/main/java/io/cloudslang/lang/runtime/steps/ExecutableExecutionData.java:199]: just rename the initial `nodeName`

**Pull request merged**